### ISSUE TITLE
Community engagement --> Product development

### ISSUE THREAD

**Issue by lukefretwell** [GetDKAN/ops#55]
Develop/deploy a plan to engage with the community on product development.

Doc: https://docs.google.com/document/d/1Nsx_kZ32WtzgIsttOlbjZZmOAJ7bNx5dofZyPfSSwuU/edit?usp=sharing

**Comment by fmizzell** [comment]
1. Having a visible product roadman
2. Having mechanisms for the community to give feedback on the roadmap
3. Having a clear governance around roadmap management
4. Facilitating Agile cycles once interest is displayed on a specific item

Those 4 things are a good starting point to keep openess and the integrity of the codebase.

**Comment by marctjones** [comment]
I think the product roadmap should include the approximate end of life for new product development on the D7 version (i.e. no new featues), the target for having a initial release, betas, and production ready version of the D8 release.

That way 1) we communicate the long term lifecycle to potential adopters and we are starting to give us a framework to talk about how much we want to invest in the current version, verse putting future enhancements into the D8 version.

**Comment by bogilvie88** [comment]
There are a reported 200+ DKAN instances in use world wide.  I would love this launch to reach those users and inform them that there is a support option, a maintainer that is in it for the long haul, and the things that Gerardo mentions above.

**Comment by nerdstein** [comment]
Please see the slide deck I posted to Slack. One way to engage a broader community is to make it easy to contribute code. I proposed a strategy of "experimental", "contributed", and "core" DKAN that helps move contributions forward without the heavy lift of being fully supportable.

See here: https://github.com/GetDKAN/dkan/issues/2168

**Comment by lukefretwell** [comment]
@nerdstein can you incorporate into ^^ doc where we're all starting to collaborate?

**Comment by nerdstein** [comment]
Shouldn't this be put into public documentation? Why google doc? I thought this was "open".

The ticket above achieves that. You can use the material in corresponding marketing / community collateral at your discretion.

**Comment by fmizzell** [comment]
@nerdstein I believe the google doc is just a working draft. Once we finish working through some of these ideas I believe they will be incorporated on the website or other public documentation.

**Comment by fmizzell** [comment]
@marctjones I wonder if it would be good enough to have an open ticket with a title like "DKAN to Drupal 8 transition plan". That is non-committal to a specific deadline yet it communicates the things you mentioned. I am not sure that at any point we need to stop feature development in D7, but I do think that any new development should be architected with portability in mind. Where those expectations should live so new contributors are aware and to keep ourselves accountable, I do not know.

**Comment by kimwdavidson** [comment]
This ticket's gotten a bit overcomplicated. Closing in favor of discrete tasks listed below:

Gerardo:
Having a visible product roadman Adam just published a roadmap (start) (This has been broken into component tickets.)
Having mechanisms for the community to give feedback on the roadmap (I believe this is what Luke is doing and it's critical) (new ticket here: https://github.com/GetDKAN/ops/issues/68)
Having a clear governance around roadmap management (I don't have any insight to this) (new ticket here: https://github.com/GetDKAN/ops/issues/69)
Facilitating Agile cycles once interest is displayed on a specific item (I got nothing here).
Those 4 things are a good starting point to keep openess and the integrity of the codebase.

Marc:
I think the product roadmap should include the approximate end of life for new product development on the D7 version (i.e. no new featues), the target for having a initial release, betas, and production ready version of the D8 release.

That way 1) we communicate the long term lifecycle to potential adopters and we are starting to give us a framework to talk about how much we want to invest in the current version, verse putting future enhancements into the D8 version. (new ticket here: https://github.com/GetDKAN/ops/issues/70)
